.github/workflows/ci.yml

```diff
@@ -73,6 +73,57 @@ jobs:
     name: "windows-2019"
     runs-on: windows-2019
```

Comment:
They are not exactly the same (the linter only runs in the VS2022 job), and I am planning to remove the VS2019 job in a few weeks.  It will be easier to remove it if it is separate, so I would prefer to keep it that way.